src/Events/Integrations/Plugins/TEC_Tweaks_Extension/Provider.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace TEC\Events\Integrations\Plugins\TEC_Tweaks_Extension;
+
+use TEC\Events\Integrations\Integration_Abstract;
+use TEC\Common\Integrations\Traits\Plugin_Integration;
+use Tribe__Events__Main;
+
+/**
+ * Class Provider
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Integrations\Plugins\TEC_Tweaks_Extension
+ */
+class Provider extends Integration_Abstract {
+	use Plugin_Integration;
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function get_slug(): string {
+		return 'tribe-ext-tec-tweaks';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function load_conditionals(): bool {
+		return class_exists( \Tribe\Extensions\Tec_Tweaks\Main::class );
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	protected function load(): void {
+		$this->remove_end_time_extension_settings();
+	}
+
+	/**
+	 * This handles removing the end time extension settings from the tweaks extension.
+	 */
+	public function remove_end_time_extension_settings() {
+		apply_filters(
+			"tribe_get_option_tribe_ext_tec_tweaks_remove_event_end_time",
+			function () {
+				return [];
+			}
+		);
```

Comment:
Apply filters? And use [`__return_empty_array`](https://developer.wordpress.org/reference/functions/__return_empty_array/)